Sources/MapboxMaps/Style/Style+Localization.swift

```diff
@@ -4,16 +4,29 @@
 extension Style {
 
     /// This function creates an expression that will localize the `textField` property of a `SymbolLayer`
-    /// - Parameter locale: A `SupportedLanguage` based `Locale`
-    internal func localizeLabels(into locale: Locale) throws {
+    /// - Parameter locale: The`Locale` to update the map to
+    /// - Parameter layerIds: An optional list of ids that need to be localized. If nothing is provided, all layers will be updated
+    public func localizeLabels(into locale: Locale, for layerIds: [String]? = nil) throws {
```

Comment:
```suggestion
    public func localizeLabels(into locale: Locale, forLayerIds layerIds: [String]? = nil) throws {
```

See "Compensate for weak type information" in the [Swift API Design Guidelines](https://swift.org/documentation/api-design-guidelines/)

Reply:
Is there a style guide or best practice we're using to determine if/when we use argument labels? I guess I'm trying to understand how I should be choosing/evaluating these two options:

`public func localizeLabels(into locale: Locale, forLayerIds layerIds: [String]? = nil) throws`

`public func localizeLabels(locale: Locale, layerIds: [String]? = nil) throws`